Documentation/ApiOverview/Events/Events/Backend/BeforeRecordDownloadIsExecutedEvent.rst

```diff
@@ -6,14 +6,6 @@
 BeforeRecordDownloadIsExecutedEvent
 ===================================
 
-..  versionadded:: 13.2
```

Comment:
I would leave the versionadded as this is a valuable information. Additionally, I would mention the removed hooks (without the migration link).

Reply:
Ok, lets do it like that